Check [WILDCARD]/test/finally_timeout.ts
running 2 tests from ./test/finally_timeout.ts
error ... FAILED ([WILDCARD])
success ... ok ([WILDCARD])

 ERRORS 

error => ./test/finally_timeout.ts:[WILDCARD]
error: Error: fail
    throw new Error("fail");
          ^
    at [WILDCARD]/test/finally_timeout.ts:4:11

 FAILURES 

error => ./test/finally_timeout.ts:[WILDCARD]

FAILED | 1 passed | 1 failed ([WILDCARD])

error: Test failed
